### DISCUSSION TITLE
send email verification

### DISCUSSION THREAD

**Discussion by ahmed-mohammed1** [filamentphp/filament#9491] [General]
i want to send email verification with login if the user not verify its email

**Comment by devhoussam1998** [comment]
https://laravel.com/docs/10.x/verification#the-email-verification-handler

**Comment by ahmed-mohammed1** [comment]
i use it and i create custom login class with override the authenticate method by adding this custom code before
checking the credentialsFromFormData  

![Uploading Screen Shot 2023-11-06 at 9.25.00 PM.png…]()
